build.yaml

```diff
@@ -0,0 +1,7 @@
+targets:
+  $default:
+    builders:
+      json_serializable:
+        options:
+          any_map: true
+          explicit_to_json: true
```

Comment:
That's the thing that I've never heard about before. I've read about that and tell me why you using it here. 😄

Reply:
yeah.. that took me a long time to find a correct phrase to google. Now I know that my json serialisable didn't cascade properly xD